src/main/java/dk/cs/aau/huppaal/abstractions/Location.java

```diff
@@ -327,12 +332,17 @@ public void deserialize(final JsonObject json) {
         setInvariantY(json.getAsJsonPrimitive(INVARIANT_Y).getAsDouble());
     }
 
-
-
     @Override
     public String generateNearString() {
-        return "Location " + (!Strings.isNullOrEmpty(getNickname()) ? (getNickname() + " (" + getId() + ")") : getId());
+        if(parent == null)
+            return generateNearStringOld();
+        return "[%s](location:%s/%s)".formatted(generateNearStringOld(), parent.getName(), getId());
     }
+
+    private String generateNearStringOld() {
+        return (!Strings.isNullOrEmpty(getNickname()) ? (getNickname() + " (" + getId() + ")") : getId());
+    }
```

Comment:
I vote for just deleting this

Reply:
removed in c88e493d2279e39df4b58fe527896ac193d4fbfa